README.md

```diff
@@ -620,11 +620,7 @@ SimpleCov.formatters = SimpleCov::Formatter::MultiFormatter.new([
 
 ## Ruby version compatibility
 
-Only Ruby 1.9+ ships with the coverage library that SimpleCov depends upon and that's what SimpleCov supports. Additionally JRuby 9.1+ is supported as well, while JRuby 1.7 and 9.0 should work they're not "officially" supported.
-SimpleCov is also built against Ruby 1.8 in [Continuous Integration], but this happens only to ensure that SimpleCov
-does not make your test suite crash right now.
-
-SimpleCov is built in [Continuous Integration] on Ruby 1.9.3, 2.0.0, 2.1, 2.2, 2.3, 2.4, 2.5 as well as JRuby 9.1.
+SimpleCov is built in [Continuous Integration] on Ruby 2.3, 2.4, 2.5 as well as JRuby 9.1 and 9.2.
```

Comment:
Sure, if we decide to stop testing against those version but not bump the `required_ruby_version` setting in the gemspec, then it could be a good note to add. I _did_ bump the setting though in this PR, since I think life is easier when you test your gem against every ruby you allow it to install against.

Reply:
Definitely, I'd tentatively also put support back in for 2+. Albeit I think required kwargs 2.1 so that's something I'd like to take along :)

Also, while many won't believe it a lot of people are running outdated rubies and I don't wanna take that storm easily :D

Reply:
Ok, so you're suggesting to put back support for 2.0, 2.1 and 2.2, but stop testing against them?

> Also, while many won't believe it a lot of people are running outdated rubies and I don't wanna take that storm easily :D

Dropping support for old rubies should break nobody's code, it will just prevent installation of newer versions of the gem on those rubies. Bundler will resolve to the newest version supporting the user's ruby version. This seems unclear so I wanted to make it explicit.

Reply:
I know it's not gonna break code, but sadly people have a history of not being too happy with it although imo it's perfectly fine for them to use older gem versions as they're the ones on an old ruby version or on an old version of a dependency. People don't tend to easily agree with me :)

I'm not suggesting putting it back in, it's my thoughts at the moment. I'd wait for feedback for @colszowka what his thoughts are. My vote currently would be 2.1+ (so that I get required kwargs) but while we drop support once we might as well just go all out and remove all EOL'ed rubies as this PR suggests. Upgrading rubies in the 2.x series is usually very easy anyhow.

Reply:
> I know it's not gonna break code, but sadly people have a history of not being too happy with it although imo it's perfectly fine for them to use older gem versions as they're the ones on an old ruby version or on an old version of a dependency. People don't tend to easily agree with me :)

Ok, I'm happy it's clear then :) I guess people have very different situations in the wild, but to me it's very weird that upgrading to the latest simplecov takes priority over stopping using ruby versions that don't get security patches applied. And even more weird to get mad at simplecov maintainers for not letting them do that :rofl:.

> I'm not suggesting putting it back in, it's my thoughts at the moment. I'd wait for feedback for @colszowka what his thoughts are. My vote currently would be 2.1+ (so that I get required kwargs) but while we drop support once we might as well just go all out and remove all EOL'ed rubies as this PR suggests. Upgrading rubies in the 2.x series is usually very easy anyhow.

Sounds good to me! Once you have a final decision I can update this PR accordingly :+1:.

Thanks!